goleft_functions.wdl

```diff
@@ -65,7 +71,7 @@ task indexcovCRAM {
 				echo "Crai file already exists with pattern *.cram.crai"
 			elif [ -f ${FILE_BASE}.crai ]; then
 				echo "Crai file already exists with pattern *.crai"
-				mv ${FILE_BASE}.crai ${FILE_BASE}.cram.crai
+				mv ${FILE_BASE}.crai ${FILE_BASE}.cram.crai  # Rename with .cram.crai pattern
```

Comment:
If the file's name is `x.cram.crai` , won't this rename it to `x.cram.cram.crai` ?

Reply:
If the input file is /cromwell-root/blah/39403943/inputs/-232903/poggers.cram
* inputCram = /cromwell-root/blah/39403943/inputs/-232903/poggers.cram
* cramBasename = poggers.cram
* prefix = poggers
* FILE_BASE = /cromwell-root/blah/39403943/inputs/-232903/poggers
* FILE_EXT = .cram

If we match ${FILE_BASE}.crai, that implies we found poggers.crai. We rename this to poggers.cram.crai for consistency's sake.